docs/features/recording.md

```diff
@@ -0,0 +1,137 @@
+
+# Room Recording (Beta)
+
+La Suite Meet offers a room recording feature that is currently in beta, with ongoing improvements planned.
+
+The feature allows users to record their room sessions. When a recording is complete, the room owner receives a notification with a link to download the recorded file. Recordings are automatically deleted after `RECORDING_EXPIRATION_DAYS`.
+
+**Current Limitations**:
```

Comment:
Are fixes planned on this (not planned yet is acceptable)? Or maybe a link to an issue for each one?